From dd138546b16a01e9c09613d02211373baf1cac02 Mon Sep 17 00:00:00 2001
From: Thomas Deutschmann <whissi@whissi.de>
Date: Thu, 20 Oct 2016 03:09:37 +0200
Subject: [PATCH] config: Fix ngx_module_type

Since commit 37182ce2f3b6 the module was not present in static builds
because due to an invalid "ngx_module_type" the module was not added to
objs/ngx_modules.c.

This commit will fix the problem by setting the correct module type
"HTTP_FILTER" [Link 1].

Link 1: https://www.nginx.com/resources/wiki/extending/new_config/#key-ngx_module_type
Gentoo-Bug: https://bugs.gentoo.org/593450
Fixes: https://github.com/masterzen/nginx-upload-progress-module/issues/50
---
 config | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config b/config
index 364af8c..5f4ade1 100644
--- a/config
+++ b/config
@@ -1,6 +1,6 @@
 ngx_addon_name=ngx_http_uploadprogress_module
 if test -n "$ngx_module_link"; then
-    ngx_module_type=FILTER
+    ngx_module_type=HTTP_FILTER
     ngx_module_name=ngx_http_uploadprogress_module
     ngx_module_srcs="$ngx_addon_dir/ngx_http_uploadprogress_module.c"
 
